python/setup.py

```diff
@@ -2,39 +2,53 @@
 from __future__ import division
 from __future__ import print_function
 
+import os
+import shutil
 import subprocess
 
-from setuptools import setup, find_packages
-import setuptools.command.install as _install
+from setuptools import setup, find_packages, Extension, Distribution
+import setuptools.command.build_ext as _build_ext
 
 
-class install(_install.install):
+class build_ext(_build_ext.build_ext):
   def run(self):
     subprocess.check_call(["../build.sh"])
-    # Calling _install.install.run(self) does not fetch required packages and
-    # instead performs an old-style install. See command/install.py in
-    # setuptools. So, calling do_egg_install() manually here.
-    self.do_egg_install()
-
-
-package_data = {
-    "ray": ["core/src/common/thirdparty/redis/src/redis-server",
-            "core/src/common/redis_module/libray_redis_module.so",
-            "core/src/plasma/plasma_store",
-            "core/src/plasma/plasma_manager",
-            "core/src/plasma/libplasma.so",
-            "core/src/local_scheduler/local_scheduler",
-            "core/src/local_scheduler/liblocal_scheduler_library.so",
-            "core/src/numbuf/libarrow.so",
```

Comment:
These were moved to `MANIFEST.in`, but note that I removed `core/src/numbuf/libarrow.so`.